olive/engine/packaging/packaging_generator.py

```diff
@@ -67,6 +69,30 @@ def _generate_zipfile_output(
         shutil.move(f"{packaging_config.name}.zip", output_dir / f"{packaging_config.name}.zip")
 
 
+def _package_models_rank(tempdir, footprints: Dict["AcceleratorSpec", "Footprint"]):
+    metrics_dict = next(iter(footprints.values())).objective_dict
```

Comment:
just curious the reason that we only get the first node metrics?

Reply:
All nodes should have the same metrics dict so getting the metrics dict from any of those should be fine.